### ISSUE TITLE
Will there be support for Forge Mod Loader?

### ISSUE THREAD

**Issue by PlasticGhoul** [JamCoreModding/right-click-harvest#30]
Hello :)

Since Pam removed the right click harvesting from all of her new versions, I am now stuck with a ton of crops I now have to farm manually.
There is currently no mod available which adds right-click-harvesting for 1.19.x - except yours.
But you mod is not available for Forge Mod Loader.

So my question is: Will there ever be a version which supports the Forge Mod Loader? Also I personally need it for MC 1.19.2 and Forge 43.2.8.

Thank you and best regards
Eric

**Comment by Jamalam360** [comment]
No, I only develop for fabric and quilt

**Comment by Jamalam360** [comment]
You're incorrect. I'm not misleading anyone, I wouldn't list a mod as Forge compatible if it wasn't. There is a newly published Forge version available on Curseforge and Modrinth as of a few weeks ago.

**Comment by Jamalam360** [comment]
There are no dependencies

Curseforge is awful at displaying it, but the Forge versions have no dependencies while the Fabric versions have some. Modrinth displays it correctly.

**Comment by Faylen** [comment]
Is modrinth really safe to use? I've seen it around often and I don't know if it's really trust worthy.

Sorry if I seemed a bit rude; Trying to make a modpack with a friend and we've come across multiple mods that either don't list dependencies anywhere and you just have to figure it out, or say they fit forge 1.19.2 while it doesn't fit forge at all, or not for 1.19.2. Seeing the problem I figured it was the same here without even testing it, and that's on me.
